CMakeLists.txt

```diff
@@ -290,10 +290,9 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 set(CMAKE_INSTALL_PREFIX ${CMAKE_CURRENT_BINARY_DIR})
 set(ARCHIVE_OUTPUT_DIRECTORY ./bin)
 
+set(CMAKE_CXX_FLAGS "-fPIC -Wall -Werror -Wuninitialized -Wvla")
 if (QNXNTO)
-  set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++11 -stdlib=libstdc++ -Wall -Werror -Wuninitialized -Wvla")
-else()
-  set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++11 -Wall -Werror -Wuninitialized -Wvla")
+  add_compile_options(-std=gnu++11 -stdlib=libstdc++)
```

Comment:
@mked-luxoft please check cmake options like `set (CMAKE_CXX_STANDARD 11)`, I would like to avoid setting compiler options in cmake if it could be done with cmake options